m4/ax_append_flag.m4

```diff
@@ -0,0 +1,50 @@
+# ===========================================================================
```

Comment:
It's easy in configure.ac to append to CFLAGS.  Adding a special m4  file seems to add obfuscation, and the code is no longer local to a single file.

Am I missing something here?